src/v2/providers/tasks.ts

```diff
@@ -144,9 +147,6 @@ export interface TaskQueueOptions extends options.EventHandlerOptions {
    * Secrets to bind to a function.
    */
   secrets?: string[];
-
-  /** Whether failed executions should be delivered again. */
-  retry?: boolean;
```

Comment:
Removal intentional?

Reply:
Thanks for asking! Yes - it's intentional. I don't think this function make sense for task queue functions; we already have `retryConfig` setup and largely ignore this config option.